src/sql/Sql.hpp

```diff
@@ -1,10 +1,25 @@
 #pragma once
+#include <vector>
+#include <string>
 
-namespace bakcyl::sql {
+namespace bakcyl::sql 
+{
 
 class Sql {
 public:
     Sql();
+    //Products queries
+    std::vector<Product> selectAllProducts();
+    Product selectProduct(std::uint64_t productId);
```

Comment:
to sie kompiluje ? Skąd w tym pliku znamy typ Product ? Nie widze zadnego includa na to